src/language_features/document_symbol.rs

```diff
@@ -258,23 +305,29 @@
         };
         let width2 = columns
             .iter()
-            .map(|(_, kind, _)| UnicodeWidthStr::width(kind.as_str()))
+            .map(|(_, description, _)| UnicodeWidthStr::width(description.as_str()))
+            .max()
+            .unwrap();
+        let width3 = columns
```

Comment:
I don't think the last column needs alignment because it's left-justified (and we don't want to add trailing spaces, right?).
With my suggestion, we can drop the alignment logic altogether, because its only use is document symbols where the filename is always the same.
For `lsp-workspace-symbol` the alignment is disabled because it might cause confusion if some file paths are really long (not sure, I never use it)

Reply:
I think we have to keep it because positions and line numbers can have different length.

Reply:
right, my bad. I was misled by the fact that we don't need UnicodeWidthStr for that but it doesn't hurt of course